### DISCUSSION TITLE
Restoration in Progress for over 10 hours

### DISCUSSION THREAD

**Discussion by Ivanzmartins** [supabase/supabase#23236] [Questions]
When i checked my database i couldn't 10 check any table, or any page related to my project.
The only other time this happened was when someone in my organization used point in time and the database also got stuck in **"Restoration in progress"**.
I already sent an email to support but haven't received any solution yet.
![Screenshot from 2024-04-24 14-41-02](https://github.com/supabase/supabase/assets/104395881/14a41e2e-bc6e-4cbe-8a75-a058ad410799)

**Comment by GaryAustin1** [comment]
Support is the only solution for this if hard refresh of browser or another browser also fails.

**Reply by Ivanzmartins** [reply]
I already sent the request for support, is there a way to prevent this from happening again? It is not the first time the project got stuck for technical issues from backup and the support have to manually unstuck the project.
But thanks for the heads up Gary!

**Reply by GaryAustin1** [reply]
Not that I know of.  I've mentioned it to them as a pretty bad annoyance that I see from users several times a week.
